File: src/main/java/github/saukiya/sxlevel/api/SXLevelAPI.java
package github.saukiya.sxlevel.api;

import github.saukiya.sxlevel.SXLevel;
import github.saukiya.sxlevel.data.ExpData;
import org.bukkit.entity.Player;

public class SXLevelAPI {
    /**
     * 获取玩家的经验数据
     *
     * @param player 玩家
     * @return ExpData 数据
     */
    public ExpData getPlayerData(Player player) {
        return SXLevel.dataManager.getPlayerLevel(player.getName()).toExpData();
    }
}

File: src/main/kotlin/top/wcpe/sxlevel/SXLevelAPI.kt
package top.wcpe.sxlevel

import github.saukiya.sxlevel.SXLevel
import top.wcpe.sxlevel.entity.PlayerLevel

/**
 * 由 WCPE 在 2022/7/24 5:08 创建
 *
 * Created by WCPE on 2022/7/24 5:08
 *
 * GitHub  : https://github.com/wcpe
 * QQ      : 1837019522
 * @author : WCPE
 * @since  : v1.1.0-alpha-dev-1
 */
object SXLevelAPI {
    @JvmStatic
    fun getPlayerLevel(playerName: String): PlayerLevel {
        return SXLevel.dataManager.getPlayerLevel(playerName)
    }
}